### PR TITLE
CDC

### PR DESCRIPTION
**Pull request by gitr0n1n**
+ Add Recording Link

### CODE REVIEW

**Review by TheEnthusiasticAs — APPROVED** [review]

**Review by TheEnthusiasticAs — APPROVED** [review]

**Review by TheEnthusiasticAs — APPROVED** [review]

**Review by developerkevin — CHANGES_REQUESTED** [review]
Everything LGTM. The only thing I ask you change is labeling this a "Core Dev Call." A CDC historically involves the status changes of an ECIP, not just a discussion. I believe "ECIP-1049 Keccak 256 Breakout Session" is the right way to label this call. Also, during CDC we coordinate with dev teams, groups, and the hard fork coordinator. 

I'm for changing it to what I mentioned above or "Keccak Informational Breakout Session" instead of mislabeling it a Core Devs Call. We don't want to confuse the audience afterall.

**Comment by gitr0n1n** [comment]
> Everything LGTM. The only thing I ask you change is labeling this a "Core Dev Call." A CDC historically involves the status changes of an ECIP, not just a discussion. I believe "ECIP-1049 Keccak 256 Breakout Session" is the right way to label this call. Also, during CDC we coordinate with dev teams, groups, and the hard fork coordinator.
> 
> I'm for changing it to what I mentioned above or "Keccak Informational Breakout Session" instead of mislabeling it a Core Devs Call. We don't want to confuse the audience afterall.

Sorry Kevin, I'm not seeing your issue here. This call was a Core Devs Call that aimed to find rough consensus for the ECIP-1049 proposal. You can review the Agenda and the Conclusion to the call here:
https://github.com/ethereumclassic/ECIPs/issues/382#issue-704136989

**Review by TheEnthusiasticAs — APPROVED** [review]

File: content/blog/2020-10-02-core-devs-call-15-ecip-1049-keccak-breakout-session/index.en.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+---
+title: "ETC Core Devs Call 15: ECIP-1049 Keccak Breakout Session"
+date: 2020-10-02
+author: r0n1n
+tags: ["announcement", "development", "education", "teams", "hardfork", "media"]
+---
+
+![ETC Core Devs Call 15 - ECIP-1049 Keccak Breakout Session](./ethereum_classic_ecip_wallpaper.png)
```
Comment (developerkevin) [inline content/blog/2020-10-02-core-devs-call-15-ecip-1049-keccak-breakout-session/index.en.md:8]: Please remove line 8, "ETC Core Devs Call." This was not such a meeting.

**Review by developerkevin — DISMISSED** [review]
This is ridiculous. Adding a line in the agenda saying "to gather rough consensus" just to "fit" into the criteria of a real "ETC Core Devs Call" after you've already submitted the PR, is disingenuous. This was not a "ETC Core Devs Call" period. full stop. Adding that line after I mentioned some criteria is insincere to the audience. This was not to measure "Rough Consensus" but to measure sentiment. Core Devs Calls are important calls that must be coordinated with each team, the hardfork coordinator, and focus on changing an ECIP to a new status. 

Please remove line 8's "ETC Core Devs Call 15" and replace with "ECIP-1049 Keccak Breakout Session." This is more genuine and appropriate.

File: content/blog/2020-10-02-core-devs-call-15-ecip-1049-keccak-breakout-session/index.en.md (hunk 1 above)
Comment (gitr0n1n) [reply content/blog/2020-10-02-core-devs-call-15-ecip-1049-keccak-breakout-session/index.en.md:8]: This does not appear to be valid criticism of the PR. Another manufactured talking point by this participant.

Source: https://ethereumclassic.org/blog/2020-10-02-core-devs-call-15-ecip-1049-keccak-breakout-call

**Review by developerkevin — APPROVED** [review]
Though I don't believe this was an "honest" ETC Core Devs Call, I will approve this pull request to add the audio/video.

**Comment by stevanlohja** [comment]
I thought it was a great educational call of the ones skeptical of Keccak-256 change. Keccak-256 folks were given the spotlight and answered many questions that I felt were well substantiated.

**Comment by gitr0n1n** [comment]
> I thought it was a great educational call of the ones skeptical of Keccak-256 change. Keccak-256 folks were given the spotlight and answered many questions that I felt answers were well substantiated.

Yes this was a high value Ethereum Classic Core Developers Call. Both the proponents and opponents had a detailed and educational discussion related to ECIP-1049. People vocalized their pleasure with the call and the calls conclusion in the `#general` chat. The only complaints came from two proponents of the proposal that wanted to rush ECIP-1049 through the process without any technical review (Donald and Kevin). Both of these people did not attend the call, they are the noise and not the signal.


Many of the questions presented by the opposition to the proposal lead the propoent faction to the realization that the proposal was not ready for `Accepted` status review. The proponents of ECIP-1049, ETC Coop, suggested the `Delay` of the proposal's review to allow them time to address the unknowns discussed. An additional concern was brought up by ETC Labs about a lack of of rough consensus and the potential for a fork at this moment.

As the moderator of the call, it was wonderful to hear the community discuss these issues in a profesisonal manner. It was a pleasure to host my first Core Developers Call and to have the Ethereum Classic development community embrace the call with such enthusiasm. I thank the proposal's author, all core development teams, and the mining developers for attending the Ethereum Classic Core Developers Call 15.

**Pull request merged**